packages/components/e2e/component-objects/src/main/java/org/talend/component/list/table/Table.java

```diff
@@ -14,7 +17,7 @@
  */
 public class Table extends Component {
 
-    private static final String NAME = "Table";
+    private static final String TABLE_NAME = "Table";
```

Comment:
No need to specify `TABLE`. It's already a private constant of the `Table` class.